util/net.go

```diff
@@ -0,0 +1,35 @@
+package netutils
```

Comment:
ok. let's call it net, both the directory and package, and see if it grows with time.